doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -203,7 +203,10 @@ Matter bridge
 nRF5340 Audio
 -------------
 
-|no_changes_yet_note|
+  * Updated:
+
+    * The audio application targeting the :zephyr:board:`nrf5340dk` to use pins P1.5-to-P1.9 for the I2S interface instead of P0.13-to-P0.17.
```

Comment:
```suggestion
    * The audio application targeting the :zephyr:board:`nrf5340dk` to use pins **P1.5** to **P1.9** for the I2S interface instead of **P0.13** to **P0.17**.
```
Sorry, I forgot the pins need to be bolded.

Reply:
No worries.

Quick question on the known issues: When a release is made how are the effected versions lists for still effected issues updated to include this new release if unresolved?